### PR TITLE
Add bash tags to You Hunger patches

### PR DESCRIPTION
**Pull request by sean-kang**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -7724,6 +7724,12 @@ plugins:
         subs: [ 'Falskaar' ]
         condition: 'active("Falskaar.esm") and not active("You Hunger - Falskaar.esp")'
```
Comment (MacSplody) [inline masterlist.yaml]: Plugins from the same mod don't really need a space between them, unless there are a lot of patches.

Comment (sean-kang) [reply masterlist.yaml]: Sorry. I don't get it. Do you mean the bash tags are unnecessary for the patches? Could you tell me what you meant by 'a space between them'?

Comment (MacSplody) [reply masterlist.yaml]: > Could you tell me what you meant by 'a space between them'?

The newline between the two entries.

image

Comment (sean-kang) [reply masterlist.yaml]: Okay. I removed the newline.

**Review by MacSplody — APPROVED** [review]
Perfect thanks.

**Pull request merged**